<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-27T17:56:37 IST" name="Default suite" finished-at="2024-02-27T17:56:40 IST" duration-ms="3381">
    <groups>
    </groups>
    <test started-at="2024-02-27T17:56:37 IST" name="Default test" finished-at="2024-02-27T17:56:40 IST" duration-ms="3381">
      <class name="Scripts.TextCase">
        <test-method is-config="true" signature="openapp()[pri:0, instance:Scripts.TextCase@1aa7ecca]" started-at="2024-02-27T17:56:37 IST" name="openapp" finished-at="2024-02-27T17:56:40 IST" duration-ms="3310" status="FAIL">
          <exception class="io.github.bonigarcia.wdm.config.WebDriverManagerException">
            <message>
              <![CDATA[java.lang.NumberFormatException: For input string: "public"]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.github.bonigarcia.wdm.config.WebDriverManagerException: java.lang.NumberFormatException: For input string: "public"
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:1435)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1195)
at io.github.bonigarcia.wdm.WebDriverManager.fallback(WebDriverManager.java:1451)
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:1432)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1195)
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:1425)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1195)
at io.github.bonigarcia.wdm.WebDriverManager.setup(WebDriverManager.java:427)
at GenricLib.Baseclass.openapp(Baseclass.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NumberFormatException: For input string: "public"
at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
at java.base/java.lang.Integer.parseInt(Integer.java:668)
at java.base/java.lang.Integer.parseInt(Integer.java:786)
at io.github.bonigarcia.wdm.managers.ChromeDriverManager.buildUrl(ChromeDriverManager.java:178)
at io.github.bonigarcia.wdm.managers.ChromeDriverManager.buildUrl(ChromeDriverManager.java:155)
at io.github.bonigarcia.wdm.WebDriverManager.download(WebDriverManager.java:1272)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1189)
... 39 more
]]>
            </full-stacktrace>
          </exception> <!-- io.github.bonigarcia.wdm.config.WebDriverManagerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openapp -->
        <test-method signature="Tc1()[pri:0, instance:Scripts.TextCase@1aa7ecca]" started-at="2024-02-27T17:56:40 IST" name="Tc1" finished-at="2024-02-27T17:56:40 IST" duration-ms="0" status="SKIP">
          <exception class="io.github.bonigarcia.wdm.config.WebDriverManagerException">
            <message>
              <![CDATA[java.lang.NumberFormatException: For input string: "public"]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.github.bonigarcia.wdm.config.WebDriverManagerException: java.lang.NumberFormatException: For input string: "public"
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:1435)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1195)
at io.github.bonigarcia.wdm.WebDriverManager.fallback(WebDriverManager.java:1451)
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:1432)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1195)
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:1425)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1195)
at io.github.bonigarcia.wdm.WebDriverManager.setup(WebDriverManager.java:427)
at GenricLib.Baseclass.openapp(Baseclass.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NumberFormatException: For input string: "public"
at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
at java.base/java.lang.Integer.parseInt(Integer.java:668)
at java.base/java.lang.Integer.parseInt(Integer.java:786)
at io.github.bonigarcia.wdm.managers.ChromeDriverManager.buildUrl(ChromeDriverManager.java:178)
at io.github.bonigarcia.wdm.managers.ChromeDriverManager.buildUrl(ChromeDriverManager.java:155)
at io.github.bonigarcia.wdm.WebDriverManager.download(WebDriverManager.java:1272)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1189)
... 39 more
]]>
            </full-stacktrace>
          </exception> <!-- io.github.bonigarcia.wdm.config.WebDriverManagerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Tc1 -->
        <test-method is-config="true" signature="closeapp(org.testng.ITestResult)[pri:0, instance:Scripts.TextCase@1aa7ecca]" started-at="2024-02-27T17:56:40 IST" name="closeapp" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1709036800971" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeapp -->
        <test-method is-config="true" signature="as()[pri:0, instance:Scripts.TextCase@1aa7ecca]" started-at="2024-02-27T17:56:41 IST" name="as" finished-at="2024-02-27T17:56:41 IST" duration-ms="3" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentReports.flush()" because "this.extent" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.flush()" because "this.extent" is null
at GenricLib.Baseclass.as(Baseclass.java:90)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:344)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- as -->
      </class> <!-- Scripts.TextCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
